src/main.cpp

```diff
@@ -196,6 +196,8 @@ unique_ptr<CommandTable> commands(shared_ptr<Root> root) {
                                             &RootCommands::print_object_tree_complete} },
         {"substitute",     { root_commands, &RootCommands::substitute_cmd,
                                             &RootCommands::substitute_complete} },
+        {"foreach",        { root_commands, &RootCommands::foreachCmd,
```

Comment:
Not related to this PR, but where does the mixture of camel and snake case in the RootCommands come from?

Reply:
I think we agreed on "strict" camelcase after some of the root commands already had been implemented. ("strict" meaning that camelcase also applies to these "Cmd" and "Complete" suffixes)

Reply:
Ok, just wondered if there was some kind of system behind this